cmd/internal/cli/pull.go

```diff
@@ -47,10 +48,23 @@ var (
 	keyServerURL = "https://keys.sylabs.io"
 	// unauthenticatedPull when true; wont ask to keep a unsigned container after pulling it.
 	unauthenticatedPull bool
-	// pullDir is the path that the containers will be pulled to, if set.
+	// PullDir is the path that the containers will be pulled to, if set.
```

Comment:
Why is this changed? It was correct before.

Reply:
oof - this PR has been around for long enough that a bunch of refactoring stuff has touched the same areas. Looks like I've missed an odd thing in the merge this time :-)

Reply:
Fixed in ec4f5ee